R/createmodelutils.R

```diff
@@ -5,20 +5,16 @@
 #' @return list of x, y, and weights with respective names
 #' @export
 getNamesFromOrdered <- function(use_weights, data_names) {
-  name_y_var <- data_names[1]
-  if(use_weights){
-    names_x_var <- data_names[2:length(data_names)-1]
-    name_weights_var <- data_names[length(data_names)]
-  } else {
-    names_x_var <- data_names[2:length(data_names)]
-    name_weights_var <- NULL
+  minimum_fields <- 2 + use_weights
+  assertthat::assert_that(length(data_names) >= minimum_fields)
+  y <- data_names[1]
+  x <- if (use_weights) data_names[2:length(data_names)-1] else data_names[2:length(data_names)]
```

Comment:
If the if-else statement does not fit in a single line, it is better to use the longer format.

```r
if (x) {
  doThis()
} else {
  doThat()
}
```